autorest/codegen/serializers/__init__.py

```diff
@@ -29,6 +30,31 @@
     "JinjaSerializer",
 ]
 
+def _get_environment(template_path: str) -> Environment:
+    return Environment(loader=PackageLoader("autorest.codegen", template_path))
+
+
+def _build_package_render_parameters(code_model: CodeModel) -> Any:
+    parameters = {}
+    parameters["package_name"] = code_model.options["package_name"]
+    parameters["package_version"] = code_model.options["package_version"]
+    parameters["namespace"] = code_model.options["namespace"]
+    folder_list = parameters["package_name"].split('-')
+    parameters["folder_first"] = folder_list[0]
```

Comment:
what are folder_first and folder_second? Also not sure this is the cleanest way to write the code, at the least can we model it more like the code model options, where create a new dict, instead of separately adding in key value pairs?

Reply:
I want to distinguish package render and code render. But it is also OK to use `code_model.options` directly.

Reply:
I want to distinguish package file render and SDK code render. But it is also OK to use `code_model.options` directly as long as I make fit for template files